crates/swc_atoms/Cargo.toml

```diff
@@ -28,7 +28,7 @@ rkyv      = { package = "rkyv", version = "=0.7.37", optional = true }
 rkyv-latest  = { package = "rkyv-test", version = "=0.7.38-test.2", optional = true }
 rustc-hash   = "1.1.0"
 serde        = "1"
-string_cache = "0.8.4"
+string_cache = { git = "https://github.com/swc-project/string-cache", branch = "swc" }
```

Comment:
This prevents publishing crates

Reply:
Could we make reuse the [repo](https://github.com/swc-project/string-cache) and publish a string-cache of SWC version? Something like `string-cache-swc`.

For some reason, I guess they(string-cache) just don't want to change anything. https://github.com/servo/string-cache/pull/265

Reply:
Yes, but I think you can use `patch` section of cargo to use it, and changing `string_cache` with something different is a breaking change.